pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -50,7 +50,10 @@ import (
 )
 
 const (
-	objNameHashLength = 5
+	DefaultMaxRetries        int32 = 3
+	DefaultMinBackoffSeconds int32 = 60   // 1 min
+	DefaultMaxBackoffSeconds int32 = 1800 // 30 min
```

Comment:
I think this is possible now

```suggestion
	defaultMaxRetries        int32 = 3
	defaultMinBackoffSeconds int32 = 60   // 1 min
	defaultMaxBackoffSeconds int32 = 1800 // 30 min
```

Reply:
Done